Lookup: Add a Getting Started tutorial (#3289)
DevExpress/devextreme-documentation: concepts/05 UI Components/Lookup/00 Getting Started with Lookup/35 Enable Pagination.md

> You can enable pagination of the Lookup component. This feature is intended to render the Lookup items. If the item data is remote, pagination allows to load items by parts to speed up the Lookup's working process. Follow the steps below to enable this functionality:

Reviewer: ```suggestion
Pagination allows Lookup to *render* items in batches. When data is remote, the component can also *load* items in batches if the server can partition data. Follow the steps below to enable this functionality:
```

"Allows to" is incorrect. [More on this](https://www.macmillandictionary.com/dictionary/british/allow)

1. Set the DataSource's [paginate](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#paginate) property to **true**.

> 2. Use the DataSource's [pageSize](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#pageSize) property to specify the number of visible items on one page. If you group data (see the topic before), the page size equals the amount of groups.

Reviewer: "Amount" is more like "sum".

```suggestion
2. Use the DataSource's [pageSize](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#pageSize) property to specify the number of visible items on one page. If you group data (see the topic before), the page size specifies the number of groups.
```

> If the number of items does not fit in the Lookup size, the scrolling bar appears. You can load items by parts on the "Next" button click.

Reviewer: ```suggestion

```

> 
> To do this, set the Lookup's [pageLoadMode](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#pageLoadMode) property to "nextButton". Additionally, specify the [nextButtonText](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#nextButtonText) property to rename the "Next" button.

Reviewer: ```suggestion
Lookup renders the next page when users scroll the drop-down list down to the bottom or when they click the "Next" button. To specify the desired behavior, set the Lookup's [pageLoadMode](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#pageLoadMode) property to "nextButton". Additionally, you can specify the [nextButtonText](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#nextButtonText) property if you want to rename the "Next" button.
```
